### PR TITLE
Build cleanup

### PR DESCRIPTION
**Pull request by lulf**
### Type of change

_Select the type of your PR_

- Refactoring

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

**Comment by lulf** [comment]
@enmasse-ci run tests profile=smoke

File: templates/Makefile [hunk 1]
```diff
@@ -169,6 +168,9 @@ install: ansible_install scripts_install component_install $(ENMASSE_BUNDLE) $(M
 package: prepare install
 	tar -czf build/$(INSTALLNAME).tgz -C build $(INSTALLNAME)
 
+deploy:
+	mvn deploy:deploy-file -Dfile=build/$(INSTALLNAME).tgz
```
Comment (k-wall) [inline templates/Makefile]: What is the idea of deploying this file to the remote repository?  Might this cause problems in some build environments?

File: templates/olm.enmasse.env [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # variables used in the olm yaml files
 
-VERSIONED_NAME=enmasse.${VERSION}.0
+VERSIONED_NAME=enmasse.${MAVEN_VERSION}
```
Comment (k-wall) [inline templates/olm.enmasse.env]: As VERSIONED_NAME is used as the basis of the Operator's version number, don't we need to maintain a level of indirection to suit downstream builds?

Comment (lulf) [reply templates/olm.enmasse.env]: There is already a downstream patch to set a different value for this field. This prevents the need to change an additional line in templates/csv/enmasse.package.yaml

File: templates/Makefile (hunk 1 above)
Comment (lulf) [reply templates/Makefile]: The idea is that it contains the build 'zip'. I'm not sure if we're actually using it, so I can remove it.

**Pull request merged**